src/Knp/DictionaryBundle/Command/DictionaryDumpCommand.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Knp\DictionaryBundle\Command;
+
+use Knp\DictionaryBundle\Dictionary;
+use Knp\DictionaryBundle\Dictionary\DictionaryRegistry;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Dump dictionaries with their related values
+ */
+class DictionaryDumpCommand extends Command
+{
+    protected static $defaultName = 'knp:dictionary:dump';
+
+    /**
+     * @var DictionaryRegistry $registry
+     */
+    private $registry;
+
+    public function __construct(DictionaryRegistry $registry)
+    {
+        parent::__construct();
+        $this->registry = $registry;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Dump Dictionaries')
+            ->setHelp('This command allows you to dump KNP dictionnaries and their values')
+            ->addArgument(
+                'dictionary',
+                InputArgument::OPTIONAL,
+                'Dictionary name you want to display'
+            )
+            ->setHelp(<<<EOF
+The <info>%command.name%</info> list all dictionaries with their related values.
+You can choose to list a specific dictionary:
+
+  <info>php %command.full_name% your_dictionary_name</info>
+EOF
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $dictionaryName = $input->getArgument('dictionary');
+        $io = new SymfonyStyle($input, $output);
+        $errorIo = $io->getErrorStyle();
```

Comment:
This looks useless, isn't it ?

Reply:
Only used here to style error line 72 (https://github.com/biig-io/DictionaryBundle/pull/9/files/66372a0ad0a3c562fc808893af15fad4c8c0ade9#diff-9210e16b7b40b032d49aa6e9546ab72bR72)